src/components/carousel/index.tsx

```diff
@@ -239,10 +240,8 @@ class Carousel extends Component<CarouselProps, CarouselState> {
     if (containerWidth) {
       const spacings = pageWidth === containerWidth ? 0 : this.getItemSpacings(this.props);
       const initialBreak = pageWidth - (containerWidth - pageWidth - spacings) / 2;
-      const snapToOffsets = _.times(
-        presenter.getChildrenLength(this.props),
-        index => initialBreak + index * pageWidth + this.getContainerMarginHorizontal()
-      );
+      const snapToOffsets = _.times(presenter.getChildrenLength(this.props),
```

Comment:
Is this a change or a prettify?
If it's the latter, please do it in a separate PR or at least a separate commit (preferably the first or the last).

Reply:
Prettify. I created two separate commits. One for formattings and one for actual logic change.